packages/@react-types/tabs/src/index.d.ts

```diff
@@ -28,9 +28,9 @@ export interface AriaTabProps {
   isDisabled?: boolean
 }
 
-export interface TabListProps<T> extends CollectionBase<T>, SingleSelection {}
+export interface TabListProps<T> extends CollectionBase<T>, Omit<SingleSelection, 'disallowEmptySelection'> {}
```

Comment:
Removed `disallowEmptySelection` because user can't actually specify this, it is always true

Reply:
Ah, we'll need to get this into Devon's PR then